File: core-java-practice/gcr-codebase/programming-elements/celsius_to_fahrenheit.java
import java.util.*;
public  class celsius_to_fahrenheit{
    public static void main(String[] args) {
        // Create Scanner object to take input from user
        Scanner sc=new Scanner(System.in);
        //Ask the user to enter temperature in celcius
        double celsius=sc.nextDouble();
        //convert celsius into fahrenheit using formula
        double fahren=((celsius*9/5)+32);
        System.out.println(fahren);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/CelsiusToFahrenheit.cs
using System;
class CelsiusToFahrenheit{
	public static void Main(string[] args){
		//Ask user to input temperature in celsius
		int celsius=int.Parse(Console.ReadLine());
		//covert temperature celsius to fahrenheit
		int fahr=(celsius*9/5)+32;
		//print temperature in fahrenheit
		Console.WriteLine(fahr);
	}
}